### PR TITLE
synp pr 1.0.0 beta.5

### PR DESCRIPTION
**Pull request by jenniferboedker**
sync pr

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -57,9 +57,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = '1.0.0-beta.5-SNAPSHOT'
+version = '1.0.0-beta.5'
```
Comment (Steffengreiner) [inline docs/conf.py:60]:
```suggestion
version = '1.0.0-beta.6'
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,7 +4,19 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
-1.0.0-beta.5-SNAPSHOT (2021-11-22)
+1.0.0-beta.6 (2021-12-07)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:7]: MAJOR NIT 💂:  
```suggestion
1.0.0-beta.6 (2021-12-08)
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Heya @jenniferboedker i found a major nit and one missing version adjustment. 
Once the version adjustment is done this can be approved. Good Job 👍

File: docs/conf.py (hunk 1 above)
Comment (jenniferboedker) [reply docs/conf.py:60]: It needs to be 6-SNAPSHOT, if you update your comment I can accept your suggestion :)

File: CHANGELOG.rst (hunk 2 above)
Comment (jenniferboedker) [reply CHANGELOG.rst:7]: ouh.. here I also forgot the snapshot tag..

Comment (jenniferboedker) [reply CHANGELOG.rst:7]:
```suggestion
1.0.0-beta.6-SNAPSHOT (2021-12-08)
```

**Review by Steffengreiner — APPROVED** [review]
LGTM 🚀

**Pull request merged**